.github/scripts/run_tests.py

```diff
@@ -61,7 +72,10 @@
     if e.returncode != 2:
         raise e
 
-cat = lambda x: print(open(x).read())
+
+def cat(x):
+    print(open(x).read())
```

Comment:
is it safe not to explicitly call close?

Reply:
yes